atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -196,7 +196,13 @@ def run(self):
         else:
             base_image_labels = config["Labels"] or {}
 
-        dockerfile = DockerfileParser(self.workflow.builder.df_path)
+            try:
+                base_image_env = config["Env"]
+            except KeyError:
+                self.log.debug("Parent Environment not found, not applied to Dockerfile")
+                base_image_env = {}
+
+        dockerfile = DockerfileParser(self.workflow.builder.df_path, parent_env=base_image_env)
```

Comment:
Could we wrap this in a `try: ... except:` block to catch `TypeError` and warn about using an old parser if `base_image_env` is non-empty?